### PR TITLE
Bring the changes to current version

### PR DESCRIPTION
**Pull request by GuiWar**
Thank you for contributing to the documentation.

Your Jira ticket is:
No Ticket

Have you provided changes for all the relevant versions: next, 2022.4, 2023.1, etc?
2023.1

Have you checked all new or changed links?
Yes

Is there anything else you would like us to know?
Wait Matt to review before merge.

For reference: 

- if you are an internal contributor:
  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 
- If you are an external contributor:
  - We have an [external contribution guide](../Type-of-contribution)

**This week's exciting excerpt from the style guide**
LISTS. Only use numbered lists when the sequence is important. Usually, that is a set of instructions.
For example:
1. Change the dictionary files.
2. Run genesisInstall.
3. Run remap.
In all other cases, use an **unnumbered** list. OK?

### CODE REVIEW

**Review by wjhendry — APPROVED** [review]

File: versioned_docs/version-2023.1/01_getting-started/03_go-to-the-next-level/06_forms.md [hunk 1]
```diff
@@ -201,16 +264,175 @@ public async insertTrade() {
...
+The `jsonSchema` can either be defined locally (as above) or be fetched from the server based on a resource name.
+
+```jsx
+const jsonSchemaResponse = await this.connect.getJSONSchema(this.resourceName);
+this.jsonSchema = jsonSchemaResponse?.INBOUND?.properties?.DETAILS as JSONSchema7;
+```
+
+### Custom validation
+
+When defining your `jsonSchema`, you can pass a custom validation function to any field. it is passed as `validateFn` in the `options` block.
```
Comment (matteematt) [inline versioned_docs/version-2023.1/01_getting-started/03_go-to-the-next-level/06_forms.md:380]: @wjhendry 
```suggestion
When defining your `jsonSchema`, you can pass a custom validation function to any field via the  `validateFn` key n the `options` block.
```

File: versioned_docs/version-2023.1/01_getting-started/03_go-to-the-next-level/13_charting.md [hunk 2]
```diff
@@ -58,6 +58,65 @@ Sample configuration and data for pie chart:
 
 For further configuration examples please see: [here](https://g2plot.antv.vision/en/examples/gallery).
 
+## Integration into home template
+
+You can add it into the flex layout from previous sections of the tutorial like the following:
```
Comment (matteematt) [inline versioned_docs/version-2023.1/01_getting-started/03_go-to-the-next-level/13_charting.md:63]: @wjhendry 
```suggestion
You can now integrate the chart into the existing flex layout we have made from the previous tutorial sections, and end up with the following markup:
```

**Review by matteematt — COMMENTED** [review]
I have added some suggestions where I think John has asked - he's annotated the file rather than the actual line somehow

**Review by wjhendry — APPROVED** [review]
Thanks.

**Comment by wjhendry** [comment]
@GuiWar happy for you to merge when you are ready

**Review by wjhendry — APPROVED** [review]

**Pull request merged**